com.unity.render-pipelines.high-definition/Editor/Lighting/LightUnit/LightUnitSlider.cs

```diff
@@ -148,26 +165,47 @@ GUIContent GetLightUnitTooltip(string baseTooltip, float value)
             else if (value >= 10000)
                 formatValue = (value / 1000).ToString("0.#") + "K";
             else
-                formatValue = value.ToString("#.0");
+                formatValue = value.ToString("#0.0");
 
-            string tooltip = baseTooltip + " - " + formatValue + " " + m_Descriptor.unitName;
+            var tooltip = baseTooltip + " | " + formatValue + " " + unit;
```

Comment:
This is mostly for your information, but you can use the [string interpolation syntax](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) like this: `$"{baseTooltip}|{formatValue} {unit}"`